crates/bevy_reflect/src/reflect.rs

```diff
@@ -65,6 +67,7 @@ pub enum ReflectRef<'a> {
     Set(&'a dyn Set),
     Enum(&'a dyn Enum),
     Value(&'a dyn PartialReflect),
+    Unit(&'a ()),
```

Comment:
I don't think we should introduce a new `ReflectKind` that's only used by the unit type. I feel it makes more sense to consider it an "empty tuple".

Reply:
Actually I think `()` already implements `Tuple`, so I’m not sure this even variant is even used?